packages/notifi-web-push-service-worker/lib/notifi-service-worker.ts

```diff
@@ -165,10 +202,47 @@ self.addEventListener(
 
 // ⬇ Helper functions & service worker global variables ⬇
 
-let client: NotifiFrontendClient;
-let notifiEnv: NotifiEnvironment;
 const db = createDb();
 
+async function initializeNotifiClientFromDb(): Promise<
```

Comment:
IMO, we can refactor it to take a argument of the `objectStore` Key to instanciate the frontendClient. This way, we will be able to move this method to `utils/service-worker`. And we can pass the `key` string mentioned above stored as global var to have the client object.
Even further, we might be able to also move the `getSubscription` method to `utils/service-worker`